client/src/modules/Job/pages/NoAuthLanding/components/DottedRectangle.tsx

```diff
@@ -0,0 +1,33 @@
+import { Space } from 'antd';
+import { COLORS } from '../../../theme/colors';
+
+export const DottedRectangle = () => {
+  const SIZE = new Array(40).fill(null);
+
+  return (
+    <Space size={32} wrap>
+      {SIZE.map((_, index) => (
+        <span key={index} className="dot"></span>
+      ))}
```

Comment:
Curious what kind of magic happens here

Reply:
<img width="487" alt="image" src="https://user-images.githubusercontent.com/49447404/220335691-2b3f9de5-bc8f-4250-a47f-d9ccb2d7264a.png">

Reply:
it was my first idea how to do it) maybe you can suggest another variant how to implement it

Reply:
I'd suggest not to put stubs in production code until we really need it there :)